modules/core/include/opencv2/core/matx.hpp

```diff
@@ -1002,6 +1002,8 @@ Matx<_Tp, m, n> MatxCommaInitializer<_Tp, m, n>::operator *() const
     return *dst;
 }
 
+
+
```

Comment:
Where did you get this random change?

There is nothing related on 4.x:

```
$ git diff 5.x-merge-4.x-origin..4.x -- modules/core/include/opencv2/core/matx.hpp
diff --git a/modules/core/include/opencv2/core/matx.hpp b/modules/core/include/opencv2/core/matx.hpp
index 686ff5d99b..d67168518f 100644
--- a/modules/core/include/opencv2/core/matx.hpp
+++ b/modules/core/include/opencv2/core/matx.hpp
@@ -215,7 +215,7 @@ public:
     template<int l> Matx(const Matx<_Tp, m, l>& a, const Matx<_Tp, l, n>& b, Matx_MatMulOp);
     Matx(const Matx<_Tp, n, m>& a, Matx_TOp);
 
-    _Tp val[m*n]; //< matrix elements
+    _Tp val[m*n]; ///< matrix elements
 };
 
 typedef Matx<float, 1, 2> Matx12f;
```

Reply:
https://github.com/opencv/opencv/pull/23109

Reply:
There is no new lines: https://github.com/opencv/opencv/pull/23109/files#diff-c8967625aba7b9d3e3df706034cddf3a6f97b48f76ed06a3d9ede8568fe68f25

Reply:
Got it. My fault.